pinot-core/src/test/java/com/linkedin/pinot/core/data/function/FunctionExpressionEvaluatorTest.java

```diff
@@ -24,7 +24,9 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import com.linkedin.pinot.common.request.transform.TransformExpressionTree;
```

Comment:
Only import change in the test? Would be good to add so unit tests for the new code.